packages/storybook/src/example-pages/toepassingen/aanvragen/landingspagina.ts

```diff
@@ -0,0 +1,245 @@
+              white: true,
+              content: html`
+                <div class="dso-rich-content">
+                  <h2>Vragen?</h2>
+                  <p>
+                    Heeft u vragen over uw vergunning of melding? Neem dan contact op met uw gemeente of waterschap.
+                    Heeft u vragen over hoe de website werkt? Neem dan contact op met
+                    <a href="#" class="extern">het Informatiepunt</a>.
+                  </p>
+                </div>
+              `,
+            })}
+          </div>
+        </div>
+        <div class="row dso-featured">
+          <div class="col-xs-12">
+            <h2>De stappen</h1>
+            <div class="row dso-equal-heights">
+              <div class="col-md-3 col-sm-6">
+                ${highlightBoxTemplate({
+                  white: true,
+                  dropShadow: true,
+                  step: 1,
+                  content: html`
+                    <div class="dso-rich-content">
+                      <h3>Project aanmaken</h3>
+                      <p>
+                        Maak eerst een project aan. Deze vindt u daarna terug in <a href="#">Mijn omgevingsloket</a>.
+                      </p>
+                    </div>
+                  `,
+                })}
+              </div>
+              <div class="col-md-3 col-sm-6">
+                ${highlightBoxTemplate({
+                  white: true,
+                  dropShadow: true,
+                  step: 2,
+                  content: html`
+                    <div class="dso-rich-content">
+                      <h3>Locatie</h3>
+                      <p>Kies de locatie voor uw bouwactiviteit(en).</p>
+                    </div>
+                  `,
+                })}
+              </div>
+              <div class="col-md-3 col-sm-6">
+                ${highlightBoxTemplate({
+                  white: true,
+                  dropShadow: true,
+                  step: 3,
+                  content: html`
+                    <div class="dso-rich-content">
+                      <h3>Activiteiten</h3>
+                      <p>Kies de activiteiten van uw project.</p>
+                    </div>
+                  `,
+                })}
+              </div>
+              <div class="col-md-3 col-sm-6">
+                ${highlightBoxTemplate({
+                  white: true,
+                  dropShadow: true,
+                  step: 4,
+                  content: html`
+                    <div class="dso-rich-content">
+                      <h3>Vragen beantwoorden</h3>
+                      <p>
+                        Beantwoord alle vragen die nodig zijn voor uw aanvraag. Vragen die in uw situatie niet meer
+                        relevant zijn vallen vanzelf weg.
+                      </p>
+                    </div>
+                  `,
+                })}
+              </div>
+            </div>
+            <div class="row dso-equal-heights">
```

Comment:
Hier hebben we een component voor, maar echt 'n ideale template is het niet. Ff samen naar kijken volgende week